esapi/esapi_integration_test.go

```diff
@@ -0,0 +1,39 @@
+// +build integration
+
+package esapi_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/elastic/go-elasticsearch"
+)
+
+func TestAPI(t *testing.T) {
+	t.Run("Search", func(t *testing.T) {
+		es, err := elasticsearch.NewDefaultClient()
+		if err != nil {
+			log.Fatalf("Error creating the client: %s\n", err)
```

Comment:
Use `t.Fatalf` instead of `log`. I think `log.Fatal` kills the whole runtime with `os.Exit(1)` so no other tests will execute.

Reply:
Right, this is a good catch, will fix!